app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -70,11 +71,25 @@ def authorization_params
       params.permit(OpenidConnectAuthorizeForm::ATTRS)
     end
 
+    def handle_invalid_logged_out_form
```

Comment:
Updated in 5608e6d1c622cc432d7bf2405ba29b1223756b9e to `validate_authorize_form`